### PR TITLE
Removed components rendering without crashes tests from components an…

### LINKED ISSUE #126: Remove create component without crashing tests
Each of component tests have a "creates component without crashing test" which has currently ceased to have much relevance. It would be nice to have these tests removed and not added going forward.
## Expected Behavior

Refactor all the component tests to remove the create component without crashing tests
## Current Behavior

There are create component without crashing tests
## Possible Solution

Refactor to remove the create components without crashing tests.

### PR DESCRIPTION
**Pull request by javpet**
Fixed #126 with @dhemmingson 

- Removed components "rendering without crashes" test declarations both from components and containers

### CODE REVIEW

**Review by aonomike — APPROVED** [review]
Looks great! Please review as well @SherSpock @FedericoEsparza

**Comment by yakryder** [comment]
So this suffers from the same sorts of problems as #117 -- you've got a lot of changes going in that are outside the stated scope of the PR and which aren't referenced in the commit headers or descriptions.

These are of at least four types:
- spacing changes on components
- changing single quotes to double quotes
- adding semicolons
- reformatting of shallow import statements

These changes have nothing to do with removing the tests. In addition to that, it seems to me that the mere existence of #117 as an open PR should have indicated caution about making linting changes. These changes both push the scope wider than needed and are potentially setting the stage for "last one to merge is rotten egg" -- merge headaches for whoever the last person is to get their changes in between #117 and this PR.

In short, I disagree with this work being merged in its current state. 

Happy to hear counterarguments

**Review by yakryder — DISMISSED** [review]
Please see comment in PR thread

**Comment by javpet** [comment]
I understand your arguments @SherSpock, I should have been more verbose of the changes as it makes harder for someone to review / overview what's happening. The coding style differences you mentioned I hope we can solve with the right linter & configuration, so we all stick to the same setup. Probably that should be something to pick up during the next weekend.

**Review by FedericoEsparza — COMMENTED** [review]
@javpet @SherSpock I think I'm inclined to let this one through, as it will speed up @arku 's linter PR into being merged, which will, in turn, revert all the lint changes.
